integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -63,47 +76,52 @@ function show_help() {
 # ------------------------------------------------------------------------------
 # Error print function
 function error() {
-    echo "$COLOR_RED[ERROR]: $1$COLOR_NONE"
+    echo "$COLOR_RED[ERROR]: $@$COLOR_NONE"
 }
 
 # ------------------------------------------------------------------------------
 # Info print function
 function info() {
-    echo "$COLOR_GREEN[INFO]: $1$COLOR_NONE"
+    echo "$COLOR_GREEN[INFO]: $@$COLOR_NONE"
 }
 
 # ------------------------------------------------------------------------------
 # Warning print function
 function warning() {
-    echo "$COLOR_YELLOW[WARNING]: $1$COLOR_NONE"
+    echo "$COLOR_YELLOW[WARNING]: $@$COLOR_NONE"
 }
 
 # ------------------------------------------------------------------------------
 # Show dependencies
 function show_dependencies() {
     warning "Required dependencies for Tizen SDK installation: 7z cpio unzip wget"
     warning "Required dependencies for Tizen SDK: JAVA JRE >=8.0"
+    warning "To install them on Debian or Ubuntu run: sudo apt install ${DEPENDENCIES[*]}"
 }
 
 # ------------------------------------------------------------------------------
 # Helper function for downloading packages.
 # Usage: download "url_dir_package" ${package_array[@]}
 function download() {
     echo "$COLOR_BLUE"
+    local url="$1"
+    shift
 
     PKGS=()
-    for PKG in "${@:2}"; do
+    for PKG in "${@}"; do
         PKGS+=("-A" "$PKG")
     done
 
     # Skip downloading if no packages are specified
     [[ ${#PKGS[@]} -eq 0 ]] && return
 
-    wget -r -nd --no-parent -e robots=off --progress=dot:mega "${PKGS[@]}" "$1"
+    verbose_flag="-nv"
+    [ -n "$VERBOSE" ] && verbose_flag="-v"
+    wget -r "$verbose_flag" -nd --timestamping --no-parent -e robots=off --progress=dot:mega -P "$PKG_CACHE_PATH" "${PKGS[@]}" "$url"
 
     # Check if the files have been downloaded
     for PKG in "${@:2}"; do
```

Comment:
Few lines above you've changed `"${@:2}"` to `"${@}"` bot not here.

Also, it might be work to mark `PKG` and `PKGS` as local variables (and make them lowercase).